pkgs/development/python-modules/nominal/default.nix

```diff
@@ -0,0 +1,54 @@
+{
+  lib,
+  fetchFromGitHub,
+  buildPythonPackage,
+  hatchling,
+  requests,
+  nominal-api,
+  nominal-api-protos,
+  python-dateutil,
+  conjure-python-client,
+  pandas,
+  typing-extensions,
+  click,
+  pyyaml,
+  tabulate,
+  ffmpeg-python,
+  types-tabulate,
+  polars,
+  coverage,
+}:
+buildPythonPackage rec {
+  format = "pyproject";
+  version = "1.55.0";
+  name = "nominal";
+  src = fetchFromGitHub {
+    owner = "nominal-io";
+    repo = "nominal-client";
+    rev = "537665d4dd4d0190640e7a08401e022b5e4ab2a8";
+    hash = "sha256-rWJNqCxSa///i5fl32vANmQL1U69AtY/fU6M2lYlHhU=";
+  };
```

Comment:
swap this to `fetchPypi` after https://github.com/nominal-io/nominal-client/pull/361 is merged

Reply:
Why? `fetchFromGitHub` is preferred.